### PR TITLE
Restore note about @deno-types

### PR DESCRIPTION
**Pull request by kt3k**
This note was added in https://github.com/denoland/docs/pull/502, but removed by https://github.com/denoland/docs/pull/826 (probably by mistake?)

This note addresses the question like https://github.com/denoland/deno/issues/27309#issuecomment-2535862020

### CODE REVIEW

File: runtime/reference/ts_config_migration.md [hunk 1]
```diff
@@ -279,6 +279,9 @@ The compiler hint pattern-matching is quite flexible, it accepts both quoted and
 non-quoted values for the specifier, as well as any whitespace around the equals
 sign.
```
Comment (dsherret) [inline runtime/reference/ts_config_migration.md:280]: I think we should get rid of this paragraph. People should ideally be just providing quotes.

Comment (kt3k) [reply runtime/reference/ts_config_migration.md:280]: 👍 ref https://github.com/denoland/deno_graph/pull/444

File: runtime/reference/ts_config_migration.md [hunk 2]
```diff
@@ -275,9 +275,8 @@ When you’re performing type checking on `coolLib` and using it in your file, t
 TypeScript type definitions from `coolLib.d.ts` will take precedence over
 examining the JavaScript file.
 
-The compiler hint pattern-matching is quite flexible, it accepts both quoted and
-non-quoted values for the specifier, as well as any whitespace around the equals
-sign.
+> ℹ️ _Note_: In the past the `@ts-types` directive was called `@deno-types`.
+> This alias still works, but is not recommended anymore. Use `@ts-types`.
```
Comment (thisisjofrank) [inline runtime/reference/ts_config_migration.md:279]:
```suggestion
:::note

In the past the `@ts-types` directive was called `@deno-types`.
This alias still works, but is not recommended anymore. Use `@ts-types`.

:::
```

**Review by thisisjofrank — APPROVED** [review]

File: runtime/reference/ts_config_migration.md [hunk 3]
```diff
@@ -275,9 +275,12 @@ When you’re performing type checking on `coolLib` and using it in your file, t
 TypeScript type definitions from `coolLib.d.ts` will take precedence over
 examining the JavaScript file.
 
-The compiler hint pattern-matching is quite flexible, it accepts both quoted and
-non-quoted values for the specifier, as well as any whitespace around the equals
-sign.
+:::note
+
+In the past the `@ts-types` directive was called `@deno-types`.
+This alias still works, but is not recommended anymore. Use `@ts-types`.
```
Comment (thisisjofrank) [inline runtime/reference/ts_config_migration.md:281]:
```suggestion
This alias still works, but is not recommended anymore. 
Use `@ts-types`.
```

File: runtime/reference/ts_config_migration.md [hunk 4]
```diff
@@ -275,9 +275,13 @@ When you’re performing type checking on `coolLib` and using it in your file, t
...
-The compiler hint pattern-matching is quite flexible, it accepts both quoted and
-non-quoted values for the specifier, as well as any whitespace around the equals
-sign.
+:::note
+
+In the past the `@ts-types` directive was called `@deno-types`.
+This alias still works, but is not recommended anymore. 
+Use `@ts-types`.
+
+:::
```
Comment (thisisjofrank) [inline runtime/reference/ts_config_migration.md:284]:
```suggestion
:::note

In the past the `@ts-types` directive was called `@deno-types`. This alias still
works, but is not recommended anymore. Use `@ts-types`.

:::
```

**Pull request merged**